roles/StackStorm.mongodb/defaults/main.yml

```diff
@@ -2,3 +2,27 @@
 # MongoDB version to install
 # Should be '3.2' or '3.4'
 mongodb_version: "3.4"
```

Comment:
I think that we should install 3.4 if EL7 or EL6.

Reply:
This is installing 3.4 for all platforms that ansible-st2 supports right now. The only way it would be something else is if someone set the mongodb_version var. But with bionic / EL8 etc we'll need some more intelligent version selection.

That said, selecting other versions of mongo would be a different PR.